### PR TITLE
Fetching post data from endpoint

### PR DESCRIPTION
**Pull request by gkojunior**
Used axios to fetch and post the data from the API endpoint, also displayed them on the page.

### CODE REVIEW

File: src/modules/home/components/Postings.js [hunk 1]
```diff
@@ -1,32 +1,26 @@
...
 export const Postings = () => {
+    const [post, setPost] = useState([])
+
+    useEffect(() => {
+        axios
+            .get(
+                'https://dpx3nob3b4anj5ujjhfudnsspu0ubhst.lambda-url.ap-southeast-2.on.aws/'
+            )
+            .then((res) => {
+                console.log('Getting from API ENDPOINT', res.data.posts)
```
Comment (Leigh-K) [inline src/modules/home/components/Postings.js]: remove all logs from PR

File: src/modules/home/components/Postings.js [hunk 2]
```diff
@@ -1,32 +1,26 @@
...
+                setPost(res.data.posts)
+            })
+            .catch((err) => console.log(err))
```
Comment (Leigh-K) [inline src/modules/home/components/Postings.js:17]: change the body here with a 
`TODO: // create error state for homepage`

File: src/modules/home/components/Postings.js (hunk 1 above)
Comment (gkojunior) [reply src/modules/home/components/Postings.js]: Updated!

File: src/modules/home/components/Postings.js (hunk 2 above)
Comment (gkojunior) [reply src/modules/home/components/Postings.js:17]: Hey I believe I'll need some assistance here. Confused if I should change the function in the catch or the whole line.

**Review by Leigh-K — APPROVED** [review]

**Pull request merged**